fakturogen-user/src/main/java/pl/fakturogen/user/dao/entity/UserExternalApi.java

```diff
@@ -1,4 +1,86 @@
 package pl.fakturogen.user.dao.entity;
 
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+import java.util.Objects;
+
+@Entity
+@Table(name = UserExternalApi.TABLE_NAME)
 public class UserExternalApi {
+    public static final String TABLE_NAME = "users external api";
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(name = "client_ID", nullable = false)
+    private String clientId;
+
+    @Column(name = "client_secret", nullable = false)
+    private String clientSecret;
+
+    @Column(name = "token")
+    private String accessToken;
+
+    @OneToOne
+    private User user;
+
+    public Long getId() {
+        return id;
+    }
+
+    public void setId(Long id) {
+        this.id = id;
+    }
+
+    public String getClientId() {
+        return clientId;
+    }
+
+    public void setClientId(String clientId) {
+        this.clientId = clientId;
+    }
+
+    public String getClientSecret() {
+        return clientSecret;
+    }
+
+    public void setClientSecret(String clientSecret) {
+        this.clientSecret = clientSecret;
+    }
+
+    public String getAccessToken() {
+        return accessToken;
+    }
+
+    public void setAccessToken(String accessToken) {
+        this.accessToken = accessToken;
+    }
+
+    public User getUser() {
+        return user;
+    }
+
+    public void setUser(User user) {
+        this.user = user;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+        UserExternalApi that = (UserExternalApi) o;
+        return Objects.equals(id, that.id) &&
+                Objects.equals(clientId, that.clientId);
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(id, clientId);
+    }
 }
```

Comment:
overrided toString probably will be needed in case you want to make tests with equals